pkg/resourceinterpreter/default/native/retain.go

```diff
@@ -16,8 +17,11 @@ import (
 // retentionInterpreter is the function that retains values from "observed" object.
 type retentionInterpreter func(desired *unstructured.Unstructured, observed *unstructured.Unstructured) (retained *unstructured.Unstructured, err error)
 
+const deploymentControlByHPALabel = "horizontalpodautoscaler.karmada.io/name"
```

Comment:
I prefer to move this const to  the `pkg/util/constants.go` file.

Reply:
done